editor/src/clj/editor/validation.clj

```diff
@@ -74,6 +74,13 @@
       (when-not (= (resource/type-ext v) ext)
         (format "%s '%s' is not of type %s" name (resource/resource->proj-path v) (format-ext ext)))))
 
+(defn prop-resource-comp? [v name]
```

Comment:
Weirdly, the convention for the validation function name is to reflect what would be considered an error. So for example, the validation function that will emit an error if a resource does not exist is called `prop-resource-not-exists?`, and the function that produces an error for a `nil` value is named `prop-nil?`, etc.

So I think we should rename this `prop-resource-not-component?` to follow the convention. 🤔

Reply:
yes, your option is much better. thx